tests/test_scripts.py

```diff
@@ -1016,6 +1016,66 @@ def test_load_graph():
     assert x == (1, 0), x
 
 
+def test_build_graph():
```

Comment:
Indeed, especially if I'm removing `scripts/build-graph.py`. Or moving it back into `load-graph`, which amounts to the same thing.